src/Roslyn.Suppressions/Roslyn.Suppressions/Roslyn.Suppressions.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ugras-ergun-sonarsource we don't use the legacy MSBuild project format anywhere any more. Any new project should be in the the new SDK-style format i.e. something like this: https://github.com/SonarSource/sonarlint-visualstudio/blob/5cd807f30de1e6b77b2966ab11a5bea60b98e9c4/src/Roslyn/Roslyn.Suppressions.csproj#L1